app/frontend/types.ts

```diff
@@ -208,7 +208,6 @@ export type Stakepool = {
   ticker: string
   homepage: string
   poolHash: string
-  validationError?: any
```

Comment:
Such data should not be stored together with pool data, validation (if not stored just locally), should be under a separate key, not part of the "stakePool" data itself